app/assets/javascripts/discourse/app/components/sidebar.hbs

```diff
@@ -4,9 +4,28 @@
   @class="sidebar-container"
   @scrollTop={{false}}
 >
-  <Sidebar::Sections
-    @currentUser={{this.currentUser}}
-    @collapsableSections={{true}}
-  />
+  {{#if this.showMainPanel}}
```

Comment:
Do we need a mode to also show all the panels together? In order words, we will be displaying all the sections in all panels in the sidebar.

Reply:
Technically, in that case Sections can be added to "main" panel, but yeah, option to display all of them can be added